### PR TITLE
Kooha: update to 2.2.3

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

---

Please note that I had to disable tests on this package for a few reasons. There is a PR that changes a few files that would allow tests to be run; let me know if it would be preferable to include that PR as a patch here instead of disabling the tests.

### CODE REVIEW

File: srcpkgs/Kooha/template [hunk 1]
```diff
@@ -1,22 +1,24 @@
 # Template file for 'Kooha'
 pkgname=Kooha
-version=2.0.1
+version=2.2.3
 revision=1
 build_style=meson
 build_helper="rust"
-hostmakedepends="pkg-config gettext glib-devel cargo"
-makedepends="libglib-devel gtk4-devel libadwaita-devel rust-std"
+hostmakedepends="cargo desktop-file-utils gettext glib glib-devel pkg-config"
```
Comment (abenson) [inline srcpkgs/Kooha/template:7]: Might need `gtk-update-icon-cache` in `hostmakedepends` for cross

**Comment by cinerea0** [comment]
I was doing some testing to see if I could enable encoding directly to mp4. That functionality would be provided by `gst-plugins-ugly1` and would require enabling [the gpl option](https://gitlab.freedesktop.org/gstreamer/gstreamer/-/blob/main/subprojects/gst-plugins-ugly/meson_options.txt#L21). Not sure if that would cause license problems.

File: srcpkgs/Kooha/template (hunk 1 above)
Comment (cinerea0) [reply srcpkgs/Kooha/template]: That fixed it, thanks!

File: srcpkgs/Kooha/template [hunk 2]
```diff
@@ -1,22 +1,26 @@
 # Template file for 'Kooha'
 pkgname=Kooha
-version=2.0.1
+version=2.2.3
 revision=1
 build_style=meson
 build_helper="rust"
-hostmakedepends="pkg-config gettext glib-devel cargo"
-makedepends="libglib-devel gtk4-devel libadwaita-devel rust-std"
+hostmakedepends="cargo desktop-file-utils gettext glib glib-devel
```
Comment (paper42) [inline srcpkgs/Kooha/template:7]: glib should be implied by glib-devel

Comment (cinerea0) [reply srcpkgs/Kooha/template]: Fixed in the latest push.

**Pull request merged**